python/Ganga/Core/Sandbox/Sandbox.py

```diff
@@ -56,7 +56,15 @@ def createPackedInputSandbox(sandbox_files, inws, name):
 
 #
 # Future release with tarball module
-    with open(tgzfile, 'w') as this_tarfile:
+
+    if str(tgzfile).endswith('.gz') or  str(tgzfile).endswith('.tgz'):
+        file_format = 'gz'
+    elif  str(tgzfile).endswith('.bz2'):
+        file_format = 'bz2'
+    else:
+        file_format = ''
```

Comment:
And annoyingly doesn't return ```gz``` or ```bz2```... I'll do some digging but unless I can just reduce this to a single line I'm not seeing any advantages.